charts/burrow/templates/burrow-configmap.yaml

```diff
@@ -27,18 +27,27 @@ data:
     [client-profile.{{ $key }}]
     kafka-version={{ $value.kafkaVersion | default "0.8" | quote}}
     client-id="{{ $value.clientId | default "burrow-lagchecker" }}"
+    tls="{{ $value.tls | default "burrow-mTLS" }}"
+    {{ end }}
+
+    {{ range $key, $value := .Values.burrow.config.tls }}
+    [tls.{{ $key }}]
+    certfile="{{ $value.certFile | default "/tmp/client_cert.pem" }}"
+    keyfile="{{ $value.keyFile | default "/tmp/private_key.pem" }}"
+    cafile="{{ $value.caFile | default "/tmp/truststore.pem" }}"
+    noverify={{ $value.noverify | default false }}
     {{ end }}
 
     {{ range $key, $value := .Values.burrow.config.cluster }}
-    [cluster.{{ $value.clusterName | default $key }}]
```

Comment:
Makes sense to remove 

The main reason for that was to have K8s camelCase respected as I wanted my clusters to display as `va-prod-kafka` and wanted to use camelCase in the `values.config.cluster`